app.rb

```diff
@@ -1,8 +1,29 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+
 class RockPaperScissors < Sinatra::Base
+  enable :sessions
+
   get '/test' do
     'test page'
   end
 
+  get '/' do
+    erb :index
+  end
+
+  post '/rps' do
+    @name = params[:name]
+    erb :rps
```

Comment:
Post request should be private. I would use redirect here to show the next page and display the value from name